app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    vac_errors = []
+    mask_errors = []
```

Comment:
unnecessary to use lists here, better to use only one variable for masks. b/c u can return in place of except when you catch VaccineError)